insights/terraform/modules/database/main.tf

```diff
@@ -0,0 +1,37 @@
+data "aws_caller_identity" "current" {}
+
+data "aws_s3_bucket" "moderate_bucket" {
+  bucket      = "bfd-insights-moderate-${data.aws_caller_identity.current.account_id}"
```

Comment:
The module takes in a sensitivity var, but moderate is hardcoded here.  Is this intended?

Reply:
Yes. We may refactor when a second bucket is introduced, but that is few weeks away.